locale/ja/site.json

```
@@ -106,7 +106,7 @@
     "text": "参加する",
     "code-and-learn": {
       "link": "get-involved/code-and-learn",
-      "text": "Code + Learn"
+      "text": "コード及び学習"
```

「及び」はひらがなの方がいいかもしれません。

```suggestion
      "text": "コードおよび学習"
```

JTF日本語標準スタイルガイド
https://www.jtf.jp/jp/style_guide/pdf/jtf_style_guide.pdf
> 接続詞の場合。常用漢字表にあるが、ひらがなを使う。

---

申し訳ないです，スタイルガイドの存在を知りませんでした（文章作法は日本産業規格に従っていました）。
スタイルガイドの規約に合わせます。

---

We might want to keep this as is in English